libvast/src/system/default_configuration.cpp

```diff
@@ -48,8 +48,8 @@ default_configuration::default_configuration() {
   //     deadlocked actors, eventually the downstream actors will be able to
   //     make progress and the deadlock can be resolved.
   // We do a combination of (1) and (2).
-  set("stream.max-batch-delay", caf::timespan{15ms});
-  set("scheduler.max-throughput", 500);
+  set("caf.stream.max-batch-delay", caf::timespan{15ms});
+  set("caf.scheduler.max-throughput", 500);
```

Comment:
Can you write a story for re-evaluating the CAF defaults we set with VAST after merging this PR? I think we should definitely do this before cutting the next release.

Reply:
I have written this follow-up story.